tests/end-to-end/director/api_definitions_api_test.go

```diff
@@ -43,7 +43,7 @@ func TestQuerySpecificAPIDefinition(t *testing.T) {
 				}
 			}`, actualAPI.ID, tc.gqlFieldsProvider.ForAPIDefinition()))
 	err = tc.RunOperation(context.Background(), queryAppReq, &actualAPI)
-	saveQueryInExamples(t, queryAppReq.Query(), "query api")
+	saveQueryInExamples(t, queryAppReq.Query(), "query api", "query api")
```

Comment:
Consider moving the parameter representing the example directory to some const values. In some cases, I see the same string value multiple times. When having this as a const value it will be easier to maintain (move set of examples between directories) and less error-prone when adding new examples under the same set.